### PR TITLE
Ujson

### PR DESCRIPTION
**Pull request by alastair**
Includes a replacement for jsonify which uses ujson

http://tickets.musicbrainz.org/browse/LB-45

### CODE REVIEW

File: webserver/views/api.py [hunk 1]
```diff
@@ -1,13 +1,20 @@
...
 
 api_bp = Blueprint('api', __name__)
 
+def ujsonify(*args, **kwargs):
+    """An implementation of flask's jsonify which uses ujson
+    instead of json. Doesn't have as many bells and whistles
+    (no indent/separator support).
+    """
+    return Response((ujson.dumps(dict(*args, **kwargs)), '\n'),
+                        mimetype='application/json')
```
Comment (mayhem) [inline webserver/views/api.py:18]: I might need to steal this. :)

**Comment by mayhem** [comment]
:+1:

File: webserver/views/api.py (hunk 1 above)
Comment (alastair) [inline webserver/views/api.py:18]: with code potentially being shared between 5 *brainz projects, should we try and consolidate duplicate code?

Comment (mayhem) [inline webserver/views/api.py:18]: Ideally yes, but I think it is too early when we have such a simple function as the only shared component. Once we get more, we should do this though.

**Pull request merged**